src/com/opc/opcconnectionimpl.cpp

```diff
@@ -152,6 +163,30 @@ bool COpcConnectionImpl::addGroup(const char* paGroupName, unsigned long paReqUp
   return true;
 }
 
+void COpcConnectionImpl::removeItems(const char* paGroupName){
+  if(nullptr == paGroupName){
+   DEVLOG_INFO("COpcConnectionImpl::removeItems: group name is nullptr\n");
+   return;
+  }
+  DEVLOG_INFO("removing items in COpcConnectionImpl[%s]\n", paGroupName);
+  std::vector<COPCItem *> lItems;
```

Comment:
Could you please rename `lItems `to simply `items`?
I assume `l` is for list, but using the plural should suffice and does not imply any special implementation.

Reply:
Yes. I changed it.